bundler/lib/bundler/templates/newgem/newgem.gemspec.tt

```diff
@@ -25,7 +25,9 @@ Gem::Specification.new do |spec|
   # Specify which files should be added to the gem when it is released.
   # The `git ls-files -z` loads the files in the RubyGem that have been added into git.
   spec.files = Dir.chdir(File.expand_path(__dir__)) do
-    `git ls-files -z`.split("\x0").reject { |f| f.match(%r{\A(?:test|spec|features)/}) }
+    `git ls-files -z`.split("\x0").reject do |f|
+      f.match(%r{\A(?:(?:test|spec|features)/|\.(?:git|travis|circleci)|appveyor)})
```

Comment:
```suggestion
      f.match(%r{\A(?:(?:test|spec|features)/|\.(?:github|travis|circleci)|appveyor)})
```

It seems `.github` is correctly. But If you hope to remove like `.gitignore`, It's okay to the current code.

Reply:
Yes, I intended to exclude `.gitlab-ci` also not only `.gitignore`, since 22cb599bcc considers it.

Reply:
I believe that all `.git*` files are git-related, and assume that gems don't need them to work.

Reply:
👍 I see. I agreed with your opinion.